src/orbit.jl

```diff
@@ -20,11 +20,22 @@ struct OrbitPath{T}
     z::Vector{T}
     phi::Vector{T}
     dt::Vector{T}
+    jacdets::Vector{Float64}
```

Comment:
The only thing I can think of, is that it would be nice with a comment on jacdets in the struct description to OrbitPath. Otherwise, I think I am ready to approve this pull-request.

Reply:
Also some comments on the new way of using the callbacks would be nice?

Reply:
Hi Henrik, I'll put this in now for the final update.